scss/_variables.scss

```diff
@@ -1294,18 +1294,18 @@ $dropdown-header-padding:           $spacer $spacer * .5 !default;
 // scss-docs-end dropdown-variables
 
 // scss-docs-start dropdown-dark-variables
-$dropdown-dark-color:               $gray-300 !default;
-$dropdown-dark-bg:                  $gray-800 !default;
-$dropdown-dark-border-color:        $dropdown-border-color !default;
-$dropdown-dark-divider-bg:          $dropdown-divider-bg !default;
+$dropdown-dark-color:               $white !default;
+$dropdown-dark-bg:                  $black !default;
+$dropdown-dark-border-color:        $gray-700 !default;
+$dropdown-dark-divider-bg:          $dropdown-dark-border-color !default;
 $dropdown-dark-box-shadow:          null !default;
 $dropdown-dark-link-color:          $dropdown-dark-color !default;
-$dropdown-dark-link-hover-color:    $white !default;
```

Comment:
According to the design guidelines, dark links must always have the same color, the same active color and the same color everywhere (dropdons, navbar, alerts, simple text, ect).

That's why I think we must avoid using duplicate code because creating different variables for the same color (ie, $dropdown-dark-link-color, $navbar-dark-link-color, $alert-dark-link-color for instance) will make the code difficult to maintain.

In [https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1012](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1012) I created $link-dark-color ,$link-dark-hover-color, $link-dark-article-color and $link-dark-disabled-color so we can reuse them everywhere.